### ISSUE TITLE
Content bug: console.log() is used instead of console.error()

### ISSUE THREAD

**Issue by JavascriptLearner815** [mdn/content#6117]
## What page(s) did you find the problem on?

Lots of them!
https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise
...and many more!

## Specific page section or heading?

getUserMedia: _Using the new API in older browsers_
Promise: _Chained Promises_

## What is the problem?

`console.log()` is frequently used for error handling on MDN pages, most commonly in a `try/catch` block, although `console.error()` should always be used for these cases.

## What did you expect to see?

All errors to always be handled via `console.error()`.

## Did you test this? If so, how?

No, I did not need to.

**Comment by JavascriptLearner815** [comment]
It's Jan 25...

**Comment by hamishwillee** [comment]
@JavascriptLearner815 Thanks for posting. Yes, these should be `console.error()` as "good practice". However this doesn't stop any examples "working", and it can't be automated. You are welcome to start fixing specific cases, and the label "help wanted" has been added to encourage others to do so.

**Comment by sideshowbarker** [comment]
Given that this issue has been open for more than a year without anybody picking it up and creating a PR, it seems likely it would just continue to remain open for much longer without any activity.

Therefore, I’m making a judgement call and going ahead and closing it. If it turns out that anybody feels strongly otherwise, it could be re-opened.

**Comment by hamishwillee** [comment]
@teoli2003 FYI only. Note this one. It isn't modernization and it isn't a priority, but it would be a good thing to "think about at in general improvements".

I always use `console.log()` because that is what I learned first - did not even occur to me there might be separate logs for errors until I saw this. That's why our examples using "the right approach" is kind of important.

**Comment by teoli2003** [comment]
Yes, we have a few "good practice" items around Web APIs that we should enforce. (We have others like: do not use `alert()`, avoid `innerHTML` when you can, …)

We should start by documenting them somewhere in our meta docs. That way, reviewers can be more familiar with them and pay attention. Then we can see what we can do to update our existing docs.

**Comment by hamishwillee** [comment]
I could create a community discussion if you like? But you're doing such a fine job on the modernization thread I figure you might want to?
